app/src/main/java/org/pragyan/dalal18/utils/Constants.java

```diff
@@ -5,7 +5,7 @@
     // TODO (Release): Change server IP and Port
 
     public static final String HOST = "dalal.pragyan.org";
-    public static final int PORT = 8080;
+    public static final int PORT = 8000;
```

Comment:
revert this to original configuration.

Reply:
But the server is running at Port 8000. Original port number has changed in Server